.gitlab-ci.yml

```diff
@@ -48,6 +48,7 @@ test:lint:
     - export DOCKER_TAG=${CI_COMMIT_REF_SLUG:-master}
     - export SERVICE_IMAGE=${DOCKER_REPOSITORY}:${DOCKER_TAG}
     - apk add --no-cache bash git jq wget
+    - git clone --no-tags --depth=1 --branch chore/remove-middleware-reliance --single-branch https://github.com/mzedel/integration.git
```

Comment:
Is there a reason why this exist on your fork and not in the org. repo?

Reply:
yes, [the PR](https://github.com/mendersoftware/integration/pull/2181) is not merged yet...